src/main/scala/io/sdkman/changelogs/JekaMigrations.scala

```diff
@@ -0,0 +1,23 @@
+package io.sdkman.changelogs
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+
+@ChangeLog(order = "088")
+class MicronautMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001_add_jeka_candidate",
+    author = "djeang"
+  )
+  def migration001(implicit db: MongoDatabase): Candidate = {
+    Candidate(
+      candidate = "jeka",
+      name = "Jeka",
+      description =
+        "Jeka is a build tool for Java focusing on simplicity.",
```

Comment:
```suggestion
        "JeKa is a modern build tool for Java, designed to simplify and streamline your development experience. It reshapes Java application building for a generation of developers seeking lightweight, accessible, yet powerful solutions."
```

This description is used on our website and when you run `sdk list` (as is the `name`  attribute above), so we need a bit more verbosity.